app/src/renderer/components/staking/UndelegationModal.vue

```diff
@@ -36,10 +29,13 @@
         type="number"
         placeholder="Amount"
       />
+      <p v-if="maximum > 0">
+        {{ denom }}s delegated: {{ atoms(maximum) }}
+      </p>
       <tm-form-msg
-        v-if="liquidAtoms === 0"
-        :msg="`doesn't have any ${denom}s`"
-        name="Wallet"
+        v-if="maximum === 0"
+        :msg="`don't have any delegation to this validator`"
```

Comment:
when would a user see this message?

Reply:
when you try to undelegate and you don't have any delegations to that validator